frontend/src/app/chatmenu/BlockUser.tsx

```diff
@@ -1,19 +1,76 @@
 import React from 'react'
 import './BlockUser.css'
+import '../ui/User.css'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faUnlock, faUserLock } from '@fortawesome/free-solid-svg-icons'
+
 import { ChatMenuPageProps } from './ChatMenuPage'
+import { UserData } from '../types/UserData'
 
 interface BlockUserProps {
-    // TODO: Fill out the props for your component
+    participants: UserData[]
+}
+
+// create user tile to display participants
+const UserTile = ({ user }) => {
+    const { uid, displayName, photoURL } = user
+    const alt = `Profile image for ${displayName}.`
+
+    const doBlock = () => {
+        // TODO: Implement blocking the user method
+    }
+
+    const doUnblock = () => {
+        // TODO: Implement unblocking the user method
+    }
+
+    return (
+        <div className="UserTile">
+            <div className="ProfileImage">
+                <img src={photoURL} alt={alt} className="Image" />
+            </div>
+            <div className="DisplayName">
+                <span>{displayName}</span>
+            </div>
+            <div>
+                <button type="button">
+                    <FontAwesomeIcon
+                        onClick={doBlock}
+                        icon={faUserLock}
+                        className="IconBeforeBlock"
+                    />
+                    Block
+                </button>
+                <button type="button">
+                    <FontAwesomeIcon
+                        onClick={doUnblock}
+                        icon={faUnlock}
+                        className="IconBeforeUnblock"
+                    />
+                    Unblock
+                </button>
+            </div>
+        </div>
+    )
 }
 
-// TODO: Implement your component
-// eslint-disable-next-line no-empty-pattern
-const BlockUserInner = ({}: BlockUserProps) => {
-    return <p>Grace was here!</p>
+// show the users within the chat
+const BlockUserInner = ({ participants }: BlockUserProps) => {
+    return (
+        <div className="UserRow">
+            {participants.map((user) => (
+                <UserTile key={user?.uid} user={user} />
+            ))}
+        </div>
+    )
 }
 
-// TODO: Pass your component its props
-// eslint-disable-next-line no-unused-vars
-const BlockUser = ({ chat, config }: ChatMenuPageProps) => <BlockUserInner />
+// display participants to the chat menu except the logged in user
+const BlockUser = ({ chat }: ChatMenuPageProps) => (
+    <BlockUserInner
+        participants={Object.values(chat?.participants).filter((user) => user?.uid !== chat?.for)}
+    />
+)
```

Comment:
This is looking good! Can you add some stories in Storybook to make it easier to test this component? This can be done in a future pull request. I opened an issue #313 with more details.